### PR TITLE
Begins an idea for converting from travis-ci to github actions

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - name: Project Syntax Verification
+        run: make docker/run target=lint
+
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone this git repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```
Comment (dwc0011) [inline .github/workflows/lint.yml:20]: What is this git repository? This code is different then the previous so it makes me think its a different repository

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+on:
+  pull_request:
+  workflow_call:
+
+jobs:
+  tardigradelint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone this git repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```
Comment (dwc0011) [inline .github/workflows/lint.yml:11]: See comment below that the name is the same but the code is different.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+    schedule:
+      interval: weekly
+  # Maintain dependencies for dockerfiles
+  - package-ecosystem: docker
+    directory: /
+    schedule:
+      interval: weekly
+  # Maintain dependencies for terraform
+  - package-ecosystem: terraform
+    directory: "/"
```
Comment (eemperor) [inline .github/dependabot.yml:15]: Directory here has double quotes but the previous ones don't.

File: .github/workflows/lint.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Clone this git repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Download actionlint
+        id: get_actionlint
+        run: bash <(curl -sSL https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+        shell: bash
+
+      - name: Check workflow files
+        run: ${{ steps.get_actionlint.outputs.executable }}
```
Comment (dwc0011) [inline .github/workflows/lint.yml:32]: Should this provide a version, or do you want the latest version always

File: .github/workflows/lint.yml (hunk 2 above)
Comment (lorengordon) [reply .github/workflows/lint.yml]: fixed

File: .github/dependabot.yml (hunk 3 above)
Comment (lorengordon) [reply .github/dependabot.yml]: fixed

**Review by dwc0011 — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 4 above)
Comment (lorengordon) [reply .github/workflows/lint.yml]: updated to use an action to run actionlint

**Review by eemperor — APPROVED** [review]
Adding my approval also.  😀

**Pull request merged**